dietpi/conf/mpd.conf

```diff
@@ -6,7 +6,7 @@ pid_file "/var/run/mpd/pid"
 state_file "/mnt/dietpi_userdata/.mpd_cache/state"
 sticker_file "/mnt/dietpi_userdata/.mpd_cache/sticker.sql"
 
-user "root"
+user "mpd"
```

Comment:
We just need to be aware of our users, attaching USB drives and changing the MPD paths.
They need to set permissions manually.

However, if they are manually editing the config files and locations, they should expect this.

Reply:
@Fourdee 
Jep, it's basically the same for most software titles, if data they need to access is outside of dietpi_userdata. Should be fine, if default setup works fully featured.